pkgs/by-name/cl/cloudquery/package.nix

```
@@ -0,0 +1,69 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  installShellFiles,
+  stdenvNoCC,
+  versionCheckHook,
+}:
+
+buildGoModule rec {
+  pname = "cloudquery";
+  version = "6.11.2";
+
+  src = fetchFromGitHub {
+    owner = "cloudquery";
+    repo = "cloudquery";
+    rev = "refs/tags/cli-v${version}";
```

Nit. For this you need to rebase onto master.
```suggestion
    tag = "cli-v${version}";
```